spec/models/status_spec.rb

```diff
@@ -173,6 +173,22 @@
     end
   end
 
+  describe '.not_in_filtered_languages' do
+    it 'does not include statuses in filtered languages' do
+      user = Fabricate(:user, filtered_languages: ['en'])
+      status = Fabricate(:status, language: 'en')
+
+      expect(Status.not_in_filtered_languages(user.account)).not_to include status
+    end
+
+    it 'includes status with unknown language' do
+      user = Fabricate(:user)
```

Comment:
This should have filtered_languages, because `language in ...` condition will be omitted unless the user has filtered_languages.